src/Tribe/PUE/Checker.php

```diff
@@ -397,7 +397,7 @@ public function ajax_validate_key() {
 
 				// For multisite, return the network-level siteurl ... in
 				// all other cases return the actual URL being serviced
-				$queryArgs['domain'] = is_multisite() ? $this->get_network_domain() : $_SERVER['SERVER_NAME'];
+				$queryArgs['domain'] = is_multisite() ? $this->get_network_domain() : ( defined( 'WP_SITEURL' ) ? parse_url( WP_SITEURL )['host'] : $_SERVER['SERVER_NAME'] );
```

Comment:
I can certainly do that, but please note that even [PHP 5.5 is EOL and no longer supported](http://php.net/supported-versions.php) today.

Likewise, the [Requirements page on WordPress.org](https://wordpress.org/about/requirements/) now strongly recommends PHP 5.6+, too.

I would strongly recommend to drop support for older PHP versions as soon as possible, especially in case you are stating that they are officially supported somewhere.